system-int-test/src/cmr/system_int_test/utils/variable_util.clj

```diff
@@ -169,30 +172,27 @@
                (assoc :metadata "" :deleted true))
            (dissoc concept :revision-date :created-at :transaction-id :extra-fields)))))
 
-(defn sort-expected-variables
-  "Sorts the variables using the expected default sort key."
-  [variables]
-  (sort-by identity
-           (fn [t1 t2]
-             (compare (:variable-name t1) (:variable-name t2)))
-           variables))
-
 (def variable-names-in-expected-response
-  [:concept-id :revision-id :variable-name :description :originator-id])
+  [:concept-id :revision-id :provider-id :native-id])
 
 (defn assert-variable-search
   "Verifies the variable search results"
   ([variables response]
    (assert-variable-search nil variables response))
   ([expected-hits variables response]
    (let [expected-items (->> variables
-                             sort-expected-variables
-                             (map #(select-keys % variable-names-in-expected-response)))
+                             (map #(select-keys % variable-names-in-expected-response))
+                             seq
+                             set)
```

Comment:
Consistent order is important in search results, which is why we sort results. I don't think we should comparing sets and losing the order information.